Sources/MongoSwift/ReadConcern.swift

```diff
@@ -14,85 +14,104 @@ public class ReadConcern: Codable {
         case linearizable
         /// See https://docs.mongodb.com/master/reference/read-concern-snapshot/
         case snapshot
-    }
-
-    /// A pointer to a `mongoc_read_concern_t`.
-    internal var _readConcern: OpaquePointer?
+        /// Unknown level
```

Comment:
I think this docstring could be a bit more descriptive. Something like "Any other read concern level not covered by the other cases. This case is present to provide forwards compatibility with any future read concerns which may be added to new versions of MongoDB."

Reply:
The second part about why the case is present (for forwards compatibility) is important to include also. Otherwise users will wonder why we have this last case instead of just enumerating all the possibilities. 

Also, a small style nit: make sure to end all the docstrings in periods. I know I always forget to do that...